File: python-sdk/src/foremapi/models/operations/createarticle.py
import dataclasses
from typing import Optional
from ..shared import article as shared_article


@dataclasses.dataclass
class CreateArticleRequest:
    request: Optional[shared_article.Article] = dataclasses.field(default=None, metadata={'request': { 'media_type': 'application/json' }})
    

@dataclasses.dataclass
class CreateArticleResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    

File: ts-sdk/src/sdk/models/operations/createarticle.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class CreateArticleRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request?: shared.Article;
}


export class CreateArticleResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
